docs/Projects/TAD/ADRs/0003-tad-tool.md

```diff
@@ -0,0 +1,53 @@
+# ADR-0003 Tool for Transparency of Algorithmic Decision making
+
+## Context
+
+We are considering tooling for organizations to get more grip on their algorithms.
+Tooling for, for instance bias and fairness tests, and assessments (like [IAMA](https://www.rijksoverheid.nl/documenten/rapporten/2021/02/25/impact-assessment-mensenrechten-en-algoritmes)).
+
+Transparency, we think, can be fostered by sharing reports from such a tool in a [standardized way](../Reporting-Standard/latest.md).
+
+There are several existing open source
+[tools](../Existing-Tools/Comparison/tools/) which
+we [have assessed](../Existing-Tools/Comparison/tools_comparison/).
+Some support only assessments, others already combine more features and can generate a report.
+There is however no tool
+that supports all the [requirements](../Existing-Tools/Comparison/requirements/) we have.
+
+These are our main requirements:
+
+* The tool offers A one-stop shop for all aspects of the project for all stakeholders for all tasks.
+* The tool supports a workflow to track different stages of the project, it should support lifecycle management.
+* The tool can run many technical tests on a model.
+* The tool supports filling out assessments.
+* The tool supports capturing discussion around technical tests and assessments.
+* The tool offers UI access to the above requirements.
+* The tool can save results to a system card (or cards supported by system cards, like model, metrics and assessment).
+* The tool can commit the different cards to a VCS, such as git, to allow for an audit trail.
+
+## Assumptions
+
+* Collaborating or extending another project will not give us the tool we are looking for.
+* We can reuse certain components, like the
+[plugins from AIVerify](https://github.com/IMDA-BTG/aiverify/tree/main/stock-plugins), or existing libraries,
+for technical tests.
+* The tool will use a design based on loose coupled modules. This can be done by scanning directories,
+  to gather modules that implement a certain Interface.
+* Plugins will have to implement an Interface to be picked up by the system.
+* We can, to some extend, re-use the already existing POCs or findings from these POCs.
+
+## Decision
+
+We will develop our own tool. Where possible this tool should be able to re-use certain components of other open-source
+tools.
+
+## Risks
+
+* It is a lot of work to create our own tool.
+* We may not have sufficient knowledge of existing technical tests to incorporate them successfully.
+* We may struggle to get uptake of the tool if we are not aligned with envisioned users of the tool.
+
+## Consequences
+
+We will have a tool that is tailored to our needs and we do not have to discuss the direction we take and choices we make
+with third parties outside our organization.
```

Comment:
I believe we had other wording here, right? Can't see my comment anymore

Reply:
Indeed we did, the comment is 'hidden' a bit above this discussion due to being 'outdate' (which I guess happens when using (sometimes?) when using force push)..

The last thumbs up was on this:

```suggestion
## Consequences
We can develop a solution that is tailored to the needs of our stakeholders.
```